src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -124,7 +123,7 @@ public string LastName
             {
                 if (PlayerValidation.ValidateWeight(value))
                 {
-                    throw new ArgumentException(Resources.ValidationPlayerWeight, "Weight");
+                    throw new ArgumentException(Resources.ValidationPlayerWeight, nameof(Player.Weight));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The parameter name 'Weight' is not declared in the argument list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3928)